prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,22 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+from kivy.uix.label import Label
+
+
+class DynamicLabelsApp(App):
+    message = StringProperty()
```

Comment:
I'm not sure what this `message = StringProperty()` is doing here as it seems you don't use it in the program.